authority/provisioner/azure.go

```diff
@@ -219,6 +224,16 @@ func (p *Azure) Init(config Config) (err error) {
 		return
 	}
 
+	// Initialize the x509 allow/deny policy engine
+	if p.x509Policy, err = policy.NewX509PolicyEngine(p.Options.GetX509Options()); err != nil {
+		return err
+	}
+
+	// Initialize the SSH allow/deny policy engine for host certificates
+	if p.sshHostPolicy, err = policy.NewSSHHostPolicyEngine(p.Options.GetSSHOptions()); err != nil {
+		return err
+	}
+
```

Comment:
What do you think of adding this to the controller?

Reply:
Done in c40a4d269480c3be7af5408d39ef58b0b58e838c.